### DISCUSSION TITLE
Simple information is missing from the boilerplate

### DISCUSSION THREAD

**Discussion by imthedaniels** [infinitered/ignite#2728] [General]
The ignite boilerplate is very good and I believe it is among the best available in the community, however some simple things could be added so that we can use it without headaches.

For example, it took me a while to solve the problem where when updating something on a screen, the entire application updated and returned to the home screen.

And the other point I found was about ESLint, it asks me to alphabetize the CSS, but my prettier doesn't work, what did you use to fix this? Because prettier and eslint configurations don't have separate files, is everything in package.json? This information would be of great value in choosing Ignite.

But anyway, congratulations on the great work!

**Comment by flexbox** [comment]
hey @imthedaniels 

>the problem where when updating something on a screen, the entire application updated and returned to the home screen.

do you still have the problem?

>And the other point I found was about ESLint,

with the new v10 @lindboe revamped the entire ESLint setup https://github.com/infinitered/ignite/commit/510ba189ee804dd3165c5be41e30b3160b91760e

feel free to check it 🚀
